app/updaters/shopify/product_updater.rb

```diff
@@ -1,11 +1,11 @@
 module Shopify
   class ProductUpdater
-    def initialize(spree_product_id:, product_klass: Spree::Product,
+    def initialize(spree_product:,
```

Comment:
This is also a wonderful refactor. Not a huge fan of the arg named: 'product_klass`. Mainly because there is no real way/hook in Solidus to change this.